wallet/i18n/en.json

```diff
@@ -1,6 +1,6 @@
 {
   "app_title": "Pylons Wallet",
-  "something_wrong": "Something went wrong",
+  "something_wrong": "Something went wrong, Try restarting the app.",
```

Comment:
In the Easel fix, this said "Try Restarting the Application", we should be consistent.